.github/workflows/black.yml

```diff
@@ -1,6 +1,6 @@
 # This is a basic workflow to help you get started with Actions
```

Comment:
Unnecessary comment

Reply:
i have gone through all the github flows, and removed what I felt were redundant comments (where the comment simply restates the code).

If you think I deleted anything to much... let me know.